docker/docker-bootstrap.sh

```diff
@@ -46,7 +46,7 @@ case "${1}" in
     celery --app=superset.tasks.celery_app:app beat --pidfile /tmp/celerybeat.pid -l INFO -s "${SUPERSET_HOME}"/celerybeat-schedule
     ;;
   app)
-    echo "Starting web app..."
+    echo "Starting web app on debug..."
```

Comment:
I'm probably missing something, but isn't this conditional on `FLASK_DEBUG` being set to true?

Reply:
You're right, but there's the debugger and the debug on flask
"The debugger is enabled by default when the development server is run in debug mode."

adding "debug" here is not correct, I want to differentiate between starting with Flask or gunicorn